doc/modules/decomposition.rst

```diff
@@ -686,7 +685,7 @@ faces dataset, in comparison with the PCA eigenfaces.
 
 The :attr:`init` attribute determines the initialization method applied, which
 has a great impact on the performance of the method. :class:`NMF` implements
-the method Nonnegative Double Singular Value Decomposition. NNDSVD is based on
+the method Nonnegative Double Singular Value Decomposition. NNDSVD [4]_ is based on
```

Comment:
Maybe explain why the NNDSVD init should not be used with the multiplicative updates solver at this end of this paragraph.